pkgs/development/libraries/webrtc-audio-processing/enable-powerpc.patch

```diff
@@ -0,0 +1,18 @@
+diff --git a/webrtc/typedefs.h b/webrtc/typedefs.h
+index ce1f3e8..fd39a16 100644
+--- a/webrtc/typedefs.h
++++ b/webrtc/typedefs.h
+@@ -38,6 +38,13 @@
+ #define WEBRTC_ARCH_X86
+ #define WEBRTC_ARCH_32_BITS
+ #define WEBRTC_ARCH_LITTLE_ENDIAN
++#elif defined(__powerpc64__)
++#define WEBRTC_ARCH_64_BITS
++// TODO/FIXME: detect big-endian
++#define WEBRTC_ARCH_LITTLE_ENDIAN
++#elif defined(__powerpc__)
```

Comment:
(Just from context) does this need  #define WEBRTC_ARCH_32_BITS too?

Reply:
You're right.  Hang on.